packages/mdc-slider/_mixins.scss

```diff
@@ -20,116 +20,457 @@
 // THE SOFTWARE.
 //
 
+@import "@material/feature-targeting/functions";
+@import "@material/feature-targeting/mixins";
 @import "@material/theme/functions";
 @import "@material/theme/mixins";
 @import "@material/theme/variables"; // for mdc-theme-prop-value
+@import "@material/typography/mixins";
+@import "@material/rtl/mixins";
 @import "./variables";
+@import "./keyframes";
 
 //
 // Public
 //
+@mixin mdc-slider-core-styles($query: mdc-feature-all()) {
+  $feat-structure: mdc-feature-create-target($query, structure);
+  $feat-animation: mdc-feature-create-target($query, animation);
 
-@mixin mdc-slider-highlight-color($color) {
+  // postcss-bem-linter: define slider
+
+  @include mdc-feature-targets($feat-animation) {
+    @include mdc-slider-emphasize-keyframes_;
+  }
+
+  .mdc-slider {
+    @include mdc-slider-color-accessible(secondary, $query);
+
+    &--disabled {
+      $disabled-color: #9a9a9a;
+
+      @include mdc-slider-highlight-color_($disabled-color, $query);
```

Comment:
Let's name the `$query` parameter for all mixins in this file.

`$query: $query`